### DISCUSSION TITLE
(Web) Separate "First Place Ranks" for lazer and stable

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#28479] [Ideas] [answered]
## The Idea

So lazer has been "out" for quite a while, but whenever I saw people complain about it it's mostly about stuff related to first places. Like how people gonna lose their first place on the leaderboard or how easy it is to snipe a first place due to the new scoring system and much much more.

So my suggestion is this: Just show both lazer and stable first place on the web. I think it makes sense. I mean, you can't even see lazer score on stable so seems like it makes sense. Like when someone sniped you on lazer and yet it still shows that you have the first place on stable.

Also, in this way, people who can't play lazer can still get a first place if they want to.

The idea is basically something like this:

![First place ranks for lazer and first place ranks (legacy) for stable](https://i.ibb.co.com/cTc3bB9/idea.png "First place ranks for lazer and stable")

## Some Cases For Showing The First Place

**1. Case: First place only on lazer**

Condition:
* First place on lazer.
* No score / not first place on stable.

Result: Show the score only on "First Place Rank" for lazer.

**2. Case: First place only on stable**

Condition:
* First place on stable.
* Not first place on lazer.

Result: Show the score only on "First Place Rank" for legacy / stable.

**3. Case: First place on both stable and lazer**

Condition:
* First place on lazer.
* First place on stable.

Result: Show the score on both "First Place Rank" according to its score. What I meant is to show both scores even when the scores are different (like using different mods, having better acc / score, etc).

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu-infrastructure/issues/27
